_opensearch/query-dsl/term.md

```diff
@@ -245,9 +236,9 @@ GET shakespeare/_search
 }
 ```
 
-## Terms
+## Multiple terms
 
-Use the `terms` query to search for multiple terms in the same field.
+Use the `terms` operation to search for multiple values for same query field.
```

Comment:
The end of the sentence needs a small correction. Should it be "...to search for multiple values in the same query field"?